### ISSUE TITLE
Convert to MONAI

### ISSUE THREAD

**Issue by brudfors** [Project-MONAI/MetricsReloaded#2]
It would be nice if the metrics were implemented as MONAI metrics. This would require a conversion to PyTorch and removal of dependencies on ndimage and skimage, as these are not used by MONAI. It would be even nicer if the metrics could be used a loss functions in MONAI. This would, on top of PyTorch conversion, requrie the metrics to be differentiable. I would be interested in looking at this conversion.
